.travis.yml

```diff
@@ -18,8 +18,8 @@ cache:
     - $HOME/.sbt/boot/
 before_cache:
   # Tricks to avoid unnecessary cache updates
-  - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
-  - find $HOME/.sbt -name "*.lock" -delete
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
+  - find $HOME/.sbt -name "*.lock" -print -delete
```

Comment:
why `-print`?

Reply:
It prints out the files that were deleted.  Not that important, just nice to know that it's working IMO.